OpenRA.Mods.Common/Activities/Attack.cs

```diff
@@ -69,26 +71,78 @@ public override Activity Tick(Actor self)
 			if (attackStatus.HasFlag(AttackStatus.NeedsToMove))
 				return moveActivity;
 
+			if (attackStatus.HasFlag(AttackStatus.FrozenSwitch))
+			{
+				// When shooting a non-frozen target which became frozen,
+				// this makes sure that the target line will remain even after
+				// the underlying actor is destroyed
+				self.SetTargetLine(frozenSwitchActivity.Target, Color.Red);
```

Comment:
Yeah, I'm referring to that. The red line is still present when the actor died, but the unit doesn't shoot anymore. Imho that looks a bit weird (having a red attack line, but no shooting unit).